File: src/main/java/at/hannibal2/skyhanni/utils/jsonobjects/SlayerProfitTrackerItemsJson.java
package at.hannibal2.skyhanni.utils.jsonobjects;

import at.hannibal2.skyhanni.utils.NEUInternalName;
import com.google.gson.annotations.Expose;

import java.util.List;
import java.util.Map;

public class SlayerProfitTrackerItemsJson {
    @Expose
    public Map<String, List<NEUInternalName>> slayers;
}

File: src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/SlayerProfitTrackerItemsJson.kt
package at.hannibal2.hanni.data.jsonobjects.repo

import at.hannibal2.hanni.utils.NeuInternalName
import com.google.gson.annotations.Expose

data class SlayerProfitTrackerItemsJson(
    @Expose val slayers: Map<String, List<NeuInternalName>>,
)
